youths/schema/types.py

```diff
@@ -23,11 +22,9 @@
     )
 
 
-class MembershipStatus(graphene.Enum):
-    ACTIVE = "active"
-    PENDING = "pending"
-    EXPIRED = "expired"
-    RENEWING = "renewing"
+MembershipStatusEnum = graphene.Enum.from_enum(
+    MembershipStatus, description=lambda e: e.label if e else ""
```

Comment:
one thing to keep in mind, enum descriptions will not be available at the federation gateway, only at the youth membership API itself, since when federation parses the federated schema it does not go that much into detail on each `__typename`